src/components/UserInfo/UserInfo.js

```diff
@@ -0,0 +1,152 @@
+import React, { Component } from 'react';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Snackbar from '@material-ui/core/Snackbar';
+import axios from 'axios';
+import history from '../../history';
+import UserService from '../../Service/UserService.js';
+import style from './style.scss';
+
+export default class UserInfo extends Component {
+  state = {
+    newName: '',
+    snackMessage: '',
+    vertical: 'bottom',
+    horizontal: 'center',
```

Comment:
why do you keep 'vertical' and 'horizontal' in state? do you change them somewhere? why don' t use constants directly in markup?